db/schema.rb

```diff
@@ -33,6 +33,39 @@
   add_index "authentications", ["uid", "provider"], :name => "index_authentications_on_uid_and_provider", :unique => true
   add_index "authentications", ["user_id", "provider"], :name => "index_authentications_on_user_id_and_provider", :unique => true
 
+  create_table "c_rater_feedback_items", :force => true do |t|
```

Comment:
Because we are going to need to record the answer and feedback given to students on the multiple choice questions. in the argumentation block. It might make sense to make this model not be c_rater specific. It looks like the only c_rater specific field here is the 'item_id', and that could be indirectly found by going through the answer object.